/**
 * 1. You have not concluded your merge (MERGE_HEAD exists).
 * 
 * Answer: The problem is your previous pull failed to merge automatically and went to conflict state. And the conflict wasn't resolved properly before the next pull
 * 
 * 
 */